howso/react_group.amlg

```diff
@@ -9,45 +9,43 @@
 	;	"combined_model_average_distance_contribution" (list 4.05 3.9)
 	;	"distance_contributions" (list 4.5 3.2)
 	; )
-	;
-	;parameters:
-	; features: list of label names
-	; distance_contributions : calculate and output distance contribution ratios in the output assoc
-	; familiarity_conviction_addition: default to true, calculate and output familiarity conviction of adding the specified new_cases in the output assoc
-	; familiarity_conviction_removal: default to false, calculate and output familiarity conviction of removing the specified new_cases in the output assoc
-	; kl_divergence_addition: default to false, calculate and output the KL divergence of adding the specified new_cases in the output assoc
-	; kl_divergence_removal: default to false, calculate and output the KL divergence of removing the specified new_cases in the output assoc
-	; p_value_of_addition: default is false. if true will output p value of addition
-	; p_value_of_removal: default is false. if true will output p value of removal
-	; new_cases: a list of lists of lists of values corresponding to a list of sets of feature values, where the values are ordered corresponding to
-	;			the features
-	; weight_feature: optional, default '.case_weight'.  name of feature whose values to use as case weights
-	; use_case_weights: optional, flag, if set to true will scale influence weights by each case's weight_feature weight.
-	;				   If a weight is missing, uses 1 as the weight. If unspecified, case weights will be used if the trainee has them.
 	#react_group
 	(declare
 		(assoc
 			;{type "list" values "string"}
+			;list of label names
 			features (list)
 			;{type "list" values {type "list" values {type "list"} } }
+			;a list of lists of lists of values corresponding to a list of sets of feature values, where the values are ordered corresponding to
+			;	the features
 			new_cases (list)
 			;{type "boolean"}
+			;default to true, calculate and output familiarity conviction of adding the specified new_cases in the output assoc
 			familiarity_conviction_addition (true)
 			;{type "boolean"}
+			;default to false, calculate and output familiarity conviction of removing the specified new_cases in the output assoc
 			familiarity_conviction_removal (false)
 			;{type "boolean"}
+			;default to false, calculate and output the KL divergence of adding the specified new_cases in the output assoc
```

Comment:
Where the default value is a literal/immediate value such as a string or (true)/(false), let's cut that part out of the descripition as it will be included in the API JSON automatically.

But if there is more complex behavior like "default is to use all the trained features" or something like that, please leave it. Feel free to message me about any unclear judgement calls on this

Reply:
Corrected in 6d0f612

Reply:
Addressed in d90fe64